lib/model/survey_model.dart

```diff
@@ -9,3 +13,23 @@ class SurveyModel {
     required this.coverImageUrl,
   });
 }
+
+extension SurveyMapping on Survey {
+  SurveyModel fromSurveyToSurveyModel() {
```

Comment:
we can shorten the method name by getting rid of `fromSurvey` 🤔 
```suggestion
  SurveyModel toSurveyModel() {
```